test/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_executable(test_nanorange
     basic_algorithm/sorting_ops.cpp
 
     # Constexpr tests for algorithms
+    constexpr_algorithm/misc_algorithms.cpp
```

Comment:
Following the (imprecise) naming scheme in `basic_algorithm/`, this should probably be `constexpr_algorithm/non_modifying_seq_ops.cpp`

Reply:
Done.